client/src/components/templateSelectorPage/LayoutThumbnail.js

```diff
@@ -0,0 +1,114 @@
+/**
+ *    This file contains the Layout Thumbnail Component.
+ */
+import {
+  Card, Button, Modal, Carousel, Row, Col, Container,
+} from 'react-bootstrap';
+import React, { useState } from 'react';
+import { PropTypes } from 'prop-types';
+import './LayoutThumbnail.css';
+import TemplateData from './TemplateData';
+
+const LayoutThumbnail = (props) => {
+  const { layoutOption } = props;
+  const [showModal, setShowModal] = useState(false);
+
+  const handleCloseModal = () => setShowModal(false);
+  const handleShowModal = () => setShowModal(true);
+
+  const {
+    layoutName, layoutDescription, layoutLandingImage, layoutPublicationImage, layoutTeamImage, layoutAchievementsImage,
+  } = TemplateData(layoutOption);
+
+  return (
+    <>
+      <Card border="dark" className="mt-3 layout-thumbnail-card">
+        <Card.Img
+          src={layoutPublicationImage}
+          className=" layout-thumbnail-card-image"
+        />
+        <Card.ImgOverlay>
+          <div className="layout-thumbnail-transition-content">
+            <Button variant="" className="layout-thumbnail-transition-button" onClick={handleShowModal}>
+              Preview
+            </Button>
+            <LayoutPreviewModal />
+          </div>
+        </Card.ImgOverlay>
+      </Card>
+    </>
+  );
+
+  function LayoutPreviewModal() {
```

Comment:
![image](https://user-images.githubusercontent.com/17608976/137899806-8f807695-3f32-44cf-887c-cef3e0e73e36.png)
The arrow is hard to see from the above picture. Is it possible to change the arrow color to different color depends on the dark/light mode?